src/js/Router.js

```diff
@@ -25,13 +24,21 @@ export function getRoute(from = '', to = '', subway) {
         }
     }
 
+    if (!fromStation) {
```

Comment:
Error message and their positions.

Parameter from is expected: passed value is '${from}'

If expect parameters you shouldn't give them incorrect default values